environments/datasets.py

```diff
@@ -91,6 +93,20 @@ def __getitem__(self, i):
         return conversions.to_channels_first(rolls)
 
 
+class EnvironmentLoader(Dataset):
+    def __init__(self, root_dir):
+        self.root_dir = root_dir
+        self.file_list = os.listdir(root_dir)
+
+    def __len__(self):
+        return len(self.file_list)
+
+    def __getitem__(self, i):
+        rolls = np.load(os.path.join(
+            self.root_dir, self.file_list[i]))['arr_0']
+        return rolls.transpose((0, 3, 1, 2))
```

Comment:
Could you add a brief description of what does this load? From the shape it looks like a single rollout. Are the rollouts stored separately or in minibatches? And what is `arr_0` ?

Reply:
`arr_0` is a default name for the npz keys when no name is specified when saving npz files.

`__getitem__` method loads one sample of the dataset. Therefore, we load one rollout and prepare it to forward it to the network. Later, the dataloader samples `batch_size` samples and concatenates them automatically.